src/server/auth/server/admin_test.go

```diff
@@ -1661,3 +1669,142 @@ func TestGetEmptyConfig(t *testing.T) {
 	conf.LiveConfigVersion = 1 // increment version
 	require.Equal(t, conf, configResp.Configuration)
 }
+
+func TestSAMLBasic(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	deleteAll(t)
```

Comment:
Should this maybe take the client rather than creating its own? Given that you're almost always going to create one right afterward.

Reply:
The way auth works now, we actually wouldn't get that much mileage out of that change because `deleteAll` deactivates auth, which invalidates old tokens. So you'd have something like:
```
adminClient := getClient()
deleteAll(t, adminClient)
adminClient := getClient() // old adminClient token is invalid
```

Reply:
Done